src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,57 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+@Nested
+@DisplayName("String 테스트")
+public class StringTest {
+    @Test
+    void split() {
+        //요구사항1 : assertj의 contains, containsExactly 기능으로
+        String[] result = "1,2".split( ",");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    void substring() {
+        String strText = "(1,2)";
+        String[] strArr = {"(", ")"};
+        String strRslt = "";
+
+        //입력받은 문자 안에 strArr의 문자와 같은 것이 있으면 substring으로 제외
+        for (int i = 0; i < strText.length(); i++) {
+            Character cTxt = strText.charAt(i);
+            for (int j =0; j < strArr.length; j++) {
+                if (strArr[j].equals(Character.toString(cTxt))) {
+                    strRslt = strText.substring(j);
+                }
+            }
+        }
+        assertThat(strRslt).contains("1,2");
+    }
+
+
+    @Test
+    @DisplayName("문자열의 n번째 문자 가져오는 메소드")
+    void getChar() {
+        String strText = "abc";
+        int iIdx = 0;
+        String strRslt = Character.toString(strText.charAt(iIdx));
+        assertThat(strRslt).contains("a");
```

Comment:
- 문자열에 'a'가 포함되어 있는지보다는, 가져온 문자가 정확히 'a'인지를 검증하는 편이 어떨까요?
- `0` 이외의 인덱스로 나머지 문자들도 잘 가져와지는지 테스트해보면 좋을 것 같습니다.
```suggestion
        assertThat("abc".charAt(0)).isEqualTo('a');
```